core/templates/components/question-directives/question-player/question-player-concept-card-modal.controller.spec.ts

```diff
@@ -109,15 +109,17 @@ describe('Question Player Concept Card Modal Controller', function() {
     });
   }));
 
-  it('should check properties set after controller is initialized', function() {
+  it('should initialize correctly $scope properties after controller' +
+    ' initialization', function() {
     expect($scope.skillIds).toEqual(skillIds);
     expect($scope.skills).toEqual(skillsObject);
     expect($scope.index).toBe(0);
     expect($scope.modalHeader).toEqual(skillsObject[0]);
     expect($scope.isInTestMode).toBe(true);
   });
 
-  it('should go to next concept card', function() {
+  it('should change current skill concept card when going to next concept' +
```

Comment:
I think a better name here is -- 

should go to next concept card, and identify when it is the last concept card.

In general, try to include all the behaviors in the it description

Reply:
Done.